plugins/node/opentelemetry-plugin-express/src/express.ts

```diff
@@ -180,23 +180,33 @@ export class ExpressPlugin extends BasePlugin<typeof express> {
         const type = metadata.attributes[
           AttributeNames.EXPRESS_TYPE
         ] as ExpressLayerType;
-        // verify against the config if the layer should be ignored
-        if (isLayerIgnored(metadata.name, type, plugin._config)) {
-          return original.apply(this, arguments);
-        }
-        if (plugin._tracer.getCurrentSpan() === undefined) {
-          return original.apply(this, arguments);
-        }
-        // Rename the root http span once we reach the request handler
+
+        // Rename the root http span in case we haven't done it already
+        // once we reach the request handler
         if (
           metadata.attributes[AttributeNames.EXPRESS_TYPE] ===
           ExpressLayerType.REQUEST_HANDLER
         ) {
           const parent = plugin._tracer.getCurrentSpan();
-          if (parent) {
-            parent.updateName(`${req.method} ${route}`);
+          // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+          // @ts-ignore
```

Comment:
Why is this ts ignore here?

Reply:
'name' Property does not exist on the original type 'Span'. the name is added later on (in http instrumentation i think).
If i will not use ts-ignore annotation i will not pass the linter verification

Reply:
You create a new interface that extends api.Span and adds just optional name, we can't have `@ts-ignore` in here.

Reply:
@obecny no problem. done